principal_travel/static/js/script.js

```diff
@@ -102,20 +103,20 @@ var renderTable = function(tripData){
   });
 };
 
-var renderMap = function(tripData){
-  var map = L.map('map')
-  .setView([33, -8], 2);
 
+var renderMap = function(tripData){
   L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
     id: 'noonkay.f59d9773',
     // id: 'noonkay.pfg0o793',
     accessToken: 'pk.eyJ1Ijoibm9vbmtheSIsImEiOiJjaWo4cHhpa2UwMDFidXhseDg3eGMwejBuIn0.tBWxkbD9BloELWmccA1UyQ',
-    scrollwheel: false,
-    noWrap: false,
+    noWrap: true,
     minZoom: 2,
-    maxZoom: 4
+    maxZoom: 3
```

Comment:
I changed `maxZoom` to `4` because in a previous issue (#189) it was stated that we needed to increase `maxZoom` so users could better explore the map